ONSdigital/rm-collection-exercise-service
# RAS-872: Extend EQ launch claims to include sds_dataset_id if required for the form_type

# What and why?
The SDS ID is required to be passed to eQ when an SDS is present and the form type(s) in question match that supplied by SDS. This PR achieves this by grabbing the SDS attributes from the database and passing this to Fronstage for further processing.

# How to test?
1. Deploy this PR to your local environment
2. Add a CE in rOps and then add an SDS that matches the CE - this can be done by either directly adding it to the DB or adding it to the pubsub subscription 'supplementary-data-service-subscription-[YOUR-ENV]'.
3. Hit the endpoint: '/collectionexercises/[EXERCISE-ID]' of the collection exercise service
4. The SDS values should be present within the 'supplementaryDatasetJson' node of the returned JSON

Note: You can also test this when an SDS is not present. The 'supplementaryDatasetJson' will be null.

# Jira
https://jira.ons.gov.uk/browse/RAS-872
